app/post_order.py

```diff
@@ -0,0 +1,20 @@
+def postorder_traversal(root):
```

Comment:
**Code optimization Suggestion**
```python
def postorder_traversal(root):
    result = []
    def traverse(node):
        if node:
            traverse(node.left)
            traverse(node.right)
            result.append(node.value)
    traverse(root)
    return result
```
**Explanation for suggestion**
Refactored to use a single list for collecting node values instead of creating multiple lists and concatenating them, which is more efficient.